gci/views.py

```diff
@@ -13,8 +14,34 @@
     '&sp-order=-modified&sp-my_tasks=false&sp-page_size=20'
 )
 
+LOGGING_CONFIG = {
+    'version': 1,
+    'formatters': {
+        'standard': {
+            'format': '%(asctime)s [%(levelname)s] %(name)s: %(message)s'
+        }
+    },
+    'handlers': {
+        'communityHandler': {
+            'class': 'logging.FileHandler',
+            'filename': '/var/tmp/community.log',
```

Comment:
this file path should be stored in a separate constant variable.  Also this file path wont be valid on windows, so it will break any windows users.  you could simply place this file at `_site/log.txt` and it will be deployed to the live site.  not very pretty, but it is sufficient for this task.

Reply:
how about `static/community.log`?